### PR TITLE
Add 0.20 release note for PR submariner-io/cloud-prepare#1044

### PR DESCRIPTION
**Pull request by aswinsuryan**
Add 0.20 release note for PR submariner-io/cloud-prepare#1044

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ weight = 40
 * Fixed an issue with Service Discovery that caused a new `EndpointSlice` to be created when the labels on the exporting `Service`
   were updated.
 * New options were added to `subctl cloud prepare` to support a custom vpc for AWS.
+* Fixed an issue where AWS cloud prepare failed to find the VPC.
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]: I think you intended this for L16 above.

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md (hunk 1 above)
Comment (aswinsuryan) [reply src/content/community/releases/_index.en.md]: oh sorry, I got it wrong. Fixed it.

**Review by tpantelis — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Comment by dfarrell07** [comment]
@aswinsuryan needs a manual rebase

**Comment by vthapar** [comment]
@aswinsuryan Can you resolve the conflicts here?

**Pull request merged**